build.sbt

```diff
@@ -19,10 +19,15 @@ lazy val root = project.in(file("."))
 
 lazy val `scala-parser-combinators` = crossProject.in(file(".")).
   settings(scalaModuleSettings: _*).
+  jvmSettings(scalaModuleSettingsJVM).
   settings(
-    name := "scala-parser-combinators-root",
+    repoName := "scala-parser-combinators",
```

Comment:
uh, that was probably a mistake from my part, i think it should stay `name`. the plugin then sets `repoName := name.value`, which could be overridden if necessary.